udata_hydra/crawl/preprocess_check_data.py

```diff
@@ -33,7 +33,26 @@ async def preprocess_check_data(check_data: dict) -> tuple[dict, dict | None]:
 
     has_changed: bool = await has_check_changed(check_data, last_check)
     if has_changed:
-        await send_check_to_udata(check_data)
+        queue.enqueue(
+            send,
+            dataset_id=dataset_id,
+            resource_id=check_data["resource_id"],
+            document={
+                "check:available": is_valid_status(check_data.get("status")),
+                "check:status": check_data.get("status"),
+                "check:timeout": check_data["timeout"],
+                "check:date": datetime.now(timezone.utc).isoformat(),
+                "check:error": check_data.get("error"),
+                "check:headers:content-type": await get_content_type_from_header(
+                    check_data.get("headers", {})
+                ),
+                "check:headers:content-length": int(
+                    check_data.get("headers", {}).get("content-length", 0)
+                )
+                or None,
+            },
+            _priority="high",
+        )
```

Comment:
Yes, we could have leave it in its own function, but since the dedicated function was now only used to enqueue another function, that became useless factorization and was not great for readability.

Reply:
Makes sense, thanks!